cpp/CMakeLists.txt

```diff
@@ -658,6 +658,12 @@ set_source_files_properties(
   PROPERTIES COMPILE_DEFINITIONS "_FILE_OFFSET_BITS=64"
 )
 
+set_property(
```

Comment:
The line 650 above this is using `set_source_files_properties` so should we use it here?

Reply:
I'm not a CMake expert 😅. I used `set_property` since it's also used below for setting the same property for `jit/cache.cpp`.

Reply:
I'm also not cmake expert. Okay probably they both can achieve the same output :+1:

Reply:
`set_property` is probably easier to work with here because you can `APPEND` with it. This solution is fine.